osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -39,8 +41,14 @@ protected override PerformanceAttributes CreatePerformanceAttributes(ScoreInfo s
             countOk = score.Statistics.GetValueOrDefault(HitResult.Ok);
             countMeh = score.Statistics.GetValueOrDefault(HitResult.Meh);
             countMiss = score.Statistics.GetValueOrDefault(HitResult.Miss);
-            effectiveMissCount = calculateEffectiveMissCount(osuAttributes);
+            countLargeTickMiss = score.Statistics.GetValueOrDefault(HitResult.LargeTickMiss);
+            countSliderEndsDropped = score.Statistics.GetValueOrDefault(HitResult.SmallTickMiss);
+            if (!score.Mods.Any(h => h is OsuModClassic cl && cl.NoSliderHeadAccuracy.Value))
+                effectiveMissCount = countMiss; 
```

Comment:
If you can back up that claim of "almost always", I’ll consider this. I know for a fact that I have some scores that would’ve benefitted from this change had they been set in lazer, and there’s definitely some famous scores that would have benefitted as well (e.g. mou ii kai)

Reply:
literally like 99% of my nonFC scores have higher misscount in lazer compared to stable
this should be the most obvious thing ever, considering that in stable you're just ignoring sliderbreaks in misscount almost completely

Reply:
I would prefer if you could keep snide remarks such as "this should be the most obvious thing ever" out of my PRs. Insulting people will only ever detract from your suggestions. 

That said, I see what you’re saying, but I’m worried that lowering the value of a miss based on slider count will be abused by pp mappers in some form.

Reply:
okay, do the opposite thing - increase the miss penalty for not slideracc scores depending on how many sliders here

Reply:
> I would prefer if you could keep snide remarks such as "this should be the most obvious thing ever" out of my PRs. Insulting people will only ever detract from your suggestions.
> 
> That said, I see what you’re saying, but I’m worried that lowering the value of a miss based on slider count will be abused by pp mappers in some form.

also, "the most obvious thing ever" was said from the fact that you have one example (mou ii kai) of effectiveMissCount being more harsh, when ignoring the fact that there are hundreds of other examples of it being more lenient, what is kinda weird cherry-picking from your side

Reply:
> okay, i agree with this, then do the opposite thing - increase the miss penalty for not slideracc scores depending on how many sliders here
to compensate the fact that all scores with misses are overnerfed - you can decrease miss penalty

This still falls into the same trap of changing the value of a miss based on slider count. I don't want to do that. Plus, I don't feel like nerfing every stable score with misses is reasonable.

> "the most obvious thing ever" was said because of the fact that you have one example (mou ii kai) of effectiveMissCount being more harsh, while not mentioning the fact that there are hundreds of other examples of it being more lenient, what is kinda weird cherry-picking from your side

The Mou Ii Kai example was just one I pulled off the top of my head, I have plenty of my own scores for which this same idea would apply. That said, I don't think fighting anecdotes with anecdotes is particularly productive, especially when we can actually determine decently well which scores would be nerfed, which scores would be buffed, and which scores would stay the same

Scores which would be buffed
- Scores with just one miss/sb in the middle
- Scores with relatively evenly distributed misses, e.g. two misses/sb scores where one miss was ~1/3 into the map and the other was ~2/3 into the map

Scores which would stay the same
- Chokes with just one miss/sb near the end of the map
- Scores with several misses, especially with high combo, but no sbs 

Scores which would be nerfed 
- Scores with numerous sliderbreaks, more than `effectiveMissCount` can account for
- Scores with high combo, and some or many sliderbreaks
- Both cases can (and often will) also include misses

Now, I could bring up several examples of scores which would be buffed or stay the same, but I don't think the examples are what matter. None of the parameters for those categories are particularly hard to meet, and if any of the proposals that remove combo scaling are pushed, we will most definitely see more scores that fall into the buffed category. 

Regardless, many of the scores that fall into the nerfed category wouldn't be worth much regardless as that's just the nature of high misscount scores. If there's ever a reality in which this isn't true, maybe this could be revisited, but for now, I'd consider those scores to be largely negligible. 

If you know that you're going to set a score with numerous sliderbreaks, CL will be a better choice no matter what. There simply isn't a way to avoid this without being unfair to stable scores without numerous sliderbreaks. However, I think that's fine. I've outlined the scenarios in which non-CL is the same or better, and for people going for FCs or low misscounts (which are really the only scores that give pp), I feel that the insurance of knowing you won't be screwed over by sliderbreak estimations is enough to justify using it instead of CL.

Reply:
Okay, "several examples of scores"? Let's see mrekk plays (non FCs that have video or replay), and how many of them will be nerfed if they were set in lazer:
- Save me: nerfed
- Space battle: same (would be heavily nerfed if he broke slightly later)
- Owari: nerfed
- Anoyo iki: nerfed
- Yakata mawari: same
- Ghost rule: same
- Brazil: same
- Save me: nerfed
- TFaF: same
- Fur Elisa: same
- Rage of Dust: nerfed
Okay, i will stop here. I can say that there are NOT A SINGLE play in mrekk top100 that is buffed by the using true number of misses. 
Yea, this is because of the combo scaling. What about mrekk high pp scores in combo scaling removal?
- Owari 3 mod: nerfed
- Uta: same
- Yakata mawari: buffed
- Yomi Yori: nerfed
- Inai Sekai (2 miss): nerfed
- Packet Hero: nerfed
Even in your best case - scores with misses in the middle, are overnerfed by your lazer pp
Therefore your claim about "plenty of my own scores" is absolutely baseless and not based on reality (out of 17 scores - 9 are nerfed, 7 stayed the same and 1 is buffed)

Reply:
You know the situation with the slideracc? The majority of community not like this at all, and by making lazer pp that 
 harsh you're making the situation where like 95% of the people will play either stable or lazer with CL.

Reply:
>If you know that you're going to set a score with numerous sliderbreaks, CL will be a better choice no matter what. There simply isn't a way to avoid this without being unfair to stable scores without numerous sliderbreaks. However, I think that's fine. I've outlined the scenarios in which non-CL is the same or better, and for people going for FCs or low misscounts (which are really the only scores that give pp), I feel that the insurance of knowing you won't be screwed over by sliderbreak estimations is enough to justify using it instead of CL.

You're making it "CL is a better choice no matter what, except super rare edgecase", this is very bad when mod that is considered to be mod for boomers - is objectively the best way to farm for everyone

Reply:
It's lazer non-CL what should be slightly superior if anything (if equality is not possible)

Reply:
I’m just gonna resolve this since I’m not open to change on this matter for the time being. This PR is just meant to be an initial step, not a final balancing pass. We can revisit this later.